pkg/trace/agent/run.go

```diff
@@ -70,27 +56,6 @@ func Run(ctx context.Context) {
 
 	defer watchdog.LogOnPanic()
 
-	if flags.CPUProfile != "" {
```

Comment:
Are we offering CPU/Mem profiles all the time via `pprof` endpoints?

Reply:
These were never used in practice, we just use the HTTP endpoints to extract profiles whenever we want.